examples/counter/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
I’m surprised that we need that file although there is already a `.gitignore` file in the root directory. Can you double check?